### PR TITLE
style(ftux): ssh add key screen updates

### PR DESCRIPTION
**Pull request by neurosnap**
I also updated the default button styles to match spec

Screenshot 2023-04-07 at 3 01 28 PM

### CODE REVIEW

File: src/ui/shared/pre-code.tsx [hunk 1]
```diff
@@ -1,24 +1,47 @@
...
     e.preventDefault();
-    navigator.clipboard.writeText(text.join(" "));
+    navigator.clipboard.writeText(segments.map((t) => t.text).join(" "));
   };
-
-  const preTextColor = invertedColors ? "text-white" : "text-black";
-  const preStyle = invertedColors
-    ? "p-4 bg-black rounded"
-    : "p-4 bg-gray-100 rounded";
+  // const preStyle = invertedColors ? "bg-black" : "bg-gray-100";
```
Comment (madhuravius) [inline src/ui/shared/pre-code.tsx:44]: prob wanna remove this

File: src/ui/shared/pre-code.tsx [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+  (txt: string, idx: number): TextSegment => {
+    const lastElement = idx === txt.length - 1;
+    return {
+      text: txt,
+      className: lastElement ? color : highlight,
+    };
+  };
+
+export const listToInvertedTextColor = (list: string[]): TextSegment[] => {
+  return list.map(createTextColor());
```
Comment (madhuravius) [inline src/ui/shared/pre-code.tsx:20]: why bother setting optional attributes if it's so opioninated rn? just future proofing?

i say drop until needed or leave a todo/follow-up ticket

File: src/ui/shared/add-ssh-key.tsx [hunk 3]
```diff
@@ -31,49 +32,61 @@ export const AddSSHKeyForm = ({
...
-      <p className="text-gray-600 mb-2">
-        Copy the contents of the file <code>$HOME/.ssh/id_rsa.pub</code> then
-        paste here. Need Help? View Docs
-      </p>
+      <div>
+        <h4 className={tokens.type.h4}>Step 1. Copy Public SSH Key</h4>
+        <p className="text-gray-600 mb-2">
+          Copy your public key to the clipboard. If you're not sure what your
+          public key is, the following command will automatically print your SSH
+          key from ssh-agent or the default SSH directory:
```
Comment (madhuravius) [inline src/ui/shared/add-ssh-key.tsx:40]: grammar `from ssh-agent` => `from the ssh-agent` or use rephrase

**Review by madhuravius — COMMENTED** [review]
lgtm, simplified some bits, just lmk if you need reapproval post nits / unused drops (if you choose to do it)

File: src/ui/shared/pre-code.tsx (hunk 2 above)
Comment (neurosnap) [reply src/ui/shared/pre-code.tsx:20]: unintentional, i actually needed to override the colors here.  updated.

File: src/ui/shared/pre-code.tsx (hunk 1 above)
Comment (neurosnap) [reply src/ui/shared/pre-code.tsx]: done

File: src/ui/shared/add-ssh-key.tsx (hunk 3 above)
Comment (neurosnap) [reply src/ui/shared/add-ssh-key.tsx:40]: done

**Review by madhuravius — APPROVED** [review]

**Pull request merged**